src/app/style.less

```diff
@@ -0,0 +1,30 @@
+
+
+@import "variables";
+@import "mixins";
+@import "grid";
+
+@import "pages/home/home";
+
+@color: #fdd;
+
+body {
+    background: @color;
+}
+
+html, body, #app, #app-component, #home-page, #list, #map-canvas { 
```

Comment:
I have fixed this issue in a later commit. In order to keep things simple, I'd like to ignore this in this PR, as it's fixed upstream.